tests/cluster-addons/cluster.tf

```diff
@@ -0,0 +1,116 @@
+resource "kops_cluster" "cluster" {
+  name               = local.clusterName
+  admin_ssh_key      = file("${path.module}/../id_rsa.pub")
+  kubernetes_version = "1.19.12"
+  dns_zone           = local.dnsZone
+  network_id         = local.vpcId
+  cluster_addons     = [
+    file("${path.module}/kubescheduler.yaml")
+  ]
```

Comment:
LGTM! Thanks! 🎉

Reply:
Did you try it out ?
I'm concerned with this change showing perpetual diff.

Reply:
no I didn't try it -- the file content should be included in the hash though right? or why do you think it would result in a perpetual diff?

Reply:
because i'm not reading cluster addons back.